docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md

```diff
@@ -0,0 +1,211 @@
+---
+author:
+  name: Andrew Leap
+  email: andyleap@gmail.com
+description: 'Alpine Linux is a small, security oriented linux distro.  This guide details installing and configuring Alpine Linux on a KVM Linode'
+keywords: 'alpine,custom,finnix,lish,kvm'
```

Comment:
Would a user searching for "lish" want to find this article? Likewise for kvm/finnix